spec/features/search_by_food_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.describe "Food Search" do
+  it "Can search for foods by ingredient" do
+    visit root_path
+    fill_in :q, with: "sweet potatoes"
+    click_on "Search"
+    expect(current_path).to eq foods_path
+    expect(page).to have_content("Foods Found: 10")
```

Comment:
For this, we're looking to see how many results the api came back with, so you could take a look at the totalHits value that came back from your API call to get that data.